src/marketplace/ListingsView.js

```diff
@@ -32,7 +32,7 @@ class ListingsView extends React.Component<Props, State> {
 
```

Comment:
^The class `SaleObject` should be defined in the file `SaleObject.js`, not `saleObjectClass.js`